client/extensions/woocommerce/app/products/product-form-simple-card.js

```diff
@@ -60,28 +60,21 @@ const ProductFormSimpleCard = ( { product, editProduct, translate } ) => {
 		</Card>
 	);
 
-	// TODO pull in dimension and weight units.
 	const renderDeliveryDetails = () => (
 		<Card className="products__product-form-delivery-details">
 			<div className="products__product-dimensions-weight">
 				<FormFieldSet className="products__product-dimensions-input">
 					<FormLabel>{ translate( 'Dimensions' ) }</FormLabel>
 					<FormDimensionsInput
-						unit={ translate( 'in' ) }
 						dimensions={ product.dimensions }
 						onChange={ setDimension }
 					/>
 				</FormFieldSet>
 				<FormFieldSet className="products__product-weight-input">
 					<FormLabel>{ translate( 'Weight' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Weight', { context: 'Dns Record'} )` *ES Score: 19*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/woocommerce-units-display)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).